src/hooks/useInitializeWallet.ts

```diff
@@ -26,6 +25,7 @@ import { PROFILES, useExperimentalFlag } from '@/config';
 import { runKeychainIntegrityChecks } from '@/handlers/walletReadyEvents';
 import { checkPendingTransactionsOnInitialize } from '@/redux/data';
 import logger from '@/utils/logger';
+import { queryClient } from '@/react-query';
```

Comment:
not being used?

Reply:
looks like this gets removed in the other PR into this one: https://github.com/rainbow-me/rainbow/pull/5104/files#diff-53625aac5f03ab24c0c5622c32cefd9b214b500bd8131b1fcecfb01271f186b0